tetrabiodistributed/project-tetra-display
# fixed the errors that came out of the bad merge.  Closes #3

To test, run this
```bash
$ ./setup.sh
$ . venv/bin/activate
$ ./runtests.sh
```
Several tests should be skipped, but none should fail.
